### PR TITLE
feat: Support for Organization Runners

### PR DESCRIPTION
**Pull request by haridarshan**

### CODE REVIEW

File: lib/Github/Api/Organization/Actions/SelfHostedRunners.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * @link https://docs.github.com/en/rest/actions/self-hosted-runners?apiVersion=2022-11-28#list-self-hosted-runners-for-an-organization
+     *
+     * @param string $organization
+     * @param string $type
+     * @param int    $page
+     *
+     * @return array|string
+     */
+    public function all(string $organization, string $type = 'all', int $page = 1)
```
Comment (acrobat) [inline lib/Github/Api/Organization/Actions/SelfHostedRunners.php:18]: Don't add the page parameter here, if you want to paginate over all results the `ResultPager` class should be used.

Also just add a parameters array to pass the type to the call, this makes the method future proof if github decides to add more options

**Review by acrobat — CHANGES_REQUESTED** [review]
@haridarshan can you also rebase against the current master to resolve the merge conflicts? Thanks!

File: lib/Github/Api/Organization/Actions/SelfHostedRunners.php (hunk 1 above)
Comment (haridarshan) [reply lib/Github/Api/Organization/Actions/SelfHostedRunners.php]: Thanks, replaced with parameters array

File: renovate.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```
Comment (acrobat) [inline renovate.json:2]: Can you remove this unrelated file? Thanks!

**Review by acrobat — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 2 above)
Comment (haridarshan) [reply renovate.json]: Removed.

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @haridarshan! And congrats on your first contribution! 🎉
